server/views/pages/reviewPlan/checkYourAnswers/index.njk

```diff
@@ -21,12 +21,7 @@
 
       <div class="app-summary-card__change-link">
         <h2 class="govuk-heading-m app-summary-card__change-link__heading govuk-!-margin-bottom-3">
-          {% if reviewPlanDto.completedBy === 'MYSELF' %}
```

Comment:
Based on this conversation (which I forgot to add you to! Sorry!), we don't need this logic 👍 
https://moj.enterprise.slack.com/archives/C07V4CVGV5W